sharpener-admin.py

```diff
@@ -4,36 +4,46 @@
 Usage:
   sharpener-admin create_schema
   sharpener-admin populate
+  sharpener-admin start_server
 """
 
 from docopt import docopt
 from dynaconf import settings
 from google.cloud import storage
-from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine, orm
 from connectors import exercism
 from models import Base
+from api import create_app
 
 
 def create_db(db_uri):
-    engine = create_engine(db_uri, echo=True)
+    engine = create_engine(db_uri, client_encoding="utf8", echo=True)
     return Base.metadata.create_all(engine)
 
 
 def database_setup(db_uri):
-    engine = create_engine(db_uri, echo=True)
-    sess = sessionmaker(bind=engine)
+    engine = create_engine(db_uri, client_encoding="utf8", echo=True)
+    sess = orm.sessionmaker(bind=engine)
     return sess()
 
 
+session = database_setup(f"{settings.DB_API}{settings.DB_URI}")
+app = create_app(session)
+
 if __name__ == "__main__":
     args = docopt(__doc__)
     if args["create_schema"]:
-        create_db(settings.DB_API+settings.DB_URI)
+        create_db(f"{settings.DB_API}{settings.DB_URI}")
 
     if args["populate"]:
-        session = database_setup(f"{settings.DB_API}{settings.DB_URI}")
-        storage_client = storage.Client()
+        STORAGE_CLIENT = storage.Client()
```

Comment:
Why is this variable in SCREAMING_SNAKE_CASE? Don't make the poor python scream.

Reply:
Pylint was really annoying me with some nonsensical stuff. Changed to autopep8 and Pyflakes, but forgot to remove some bad style choices that it coerced me to do.